pkg/features/features.go

```diff
@@ -0,0 +1,32 @@
+package features
+
+import (
+	"k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/component-base/featuregate"
+)
+
+const (
+	DiskADController       featuregate.Feature = "DiskADController"
+	DiskDetachBeforeAttach featuregate.Feature = "DetachBeforeAttach"
+	DiskDetachBeforeDelete featuregate.Feature = "DetachBeforeDelete"
+	DBFSMetricByPlugin     featuregate.Feature = "DBFSMetricByPlugin"
+	DBFSADController       featuregate.Feature = "DBFSADController"
+)
+
+var (
+	FunctionalMutableFeatureGate = featuregate.NewFeatureGate()
+	defaultDiskFeatureGate       = map[featuregate.Feature]featuregate.FeatureSpec{
+		DiskADController:       {Default: false, PreRelease: featuregate.Alpha},
+		DiskDetachBeforeAttach: {Default: true, PreRelease: featuregate.Alpha},
+		DiskDetachBeforeDelete: {Default: true, PreRelease: featuregate.Alpha},
```

Comment:
I think these two are not suitable for feature gates. Feature gates are intended to be removed when stable. We may never remove these options.

Reply:
Ok